paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -64,6 +65,7 @@ class Paparazzi(
   private val environment: Environment = detectEnvironment(),
   private val deviceConfig: DeviceConfig = DeviceConfig.NEXUS_5,
   private val theme: String = "android:Theme.Material.NoActionBar.Fullscreen",
+  private val renderingMode: RenderingMode = RenderingMode.NORMAL,
```

Comment:
discussed offline, maybe wrap in a paparazzi type?  not sure if it matters though since it's an immutable type.

Reply:
I'm inclined to leave it exposed. Wrapping it for the sake of wrapping it seems excessive given we don't directly have any impact on its functionality.